dts/bindings/spi/atmel,sam-spi.yaml

```diff
@@ -20,3 +20,21 @@ properties:
       type: int
       description: peripheral ID
       required: true
+
+    dmas:
+      required: false
+      description: |
+        TX & RX dma specifiers.  Each specifier will have a phandle
+        reference to the dma controller, the channel number, and peripheral
+        trigger source.
```

Comment:
You need clarify how user will select the channel number and peripheral trigger. It must be clear how user select this options and where he can find the information. This info should be 100% clear without using datasheet as devicetree maintainers have been requesting.

In the case of channel number [0..23], is there a dedicated channel for each peripheral or it can be any arbitrary number? In the case of arbitrary number, is there priority involved?